src/components/sales/Order.vue

```diff
@@ -1002,7 +1006,8 @@ export default defineComponent({
         invoice_id: this.invoiceID,
         deduct_balance: this.deduct_balance
       }
-      await this.createOrder(singleOrder);
+       await this.createOrder(singleOrder);
+       this.print = true
```

Comment:
Check on button click, whether the bill should be printed or not.

Reply:
print dialog opened when order successful.